pie/utils/objects.py

```diff
@@ -294,3 +296,109 @@ async def on_timeout(self) -> None:
         """Gets called when the view timeouts."""
         self.value = None
         self.stop()
+
+
+class VotableEmbed(nextcord.Embed):
```

Comment:
Since it must be subclassed, why not add `metaclass=ABCMeta`?

https://docs.python.org/3/library/abc.html

Reply:
Looks promising, I'll try.

Reply:
Done, working.